atlp-rwanda/rca-barfoot-team3-bn
# Ft notifications after booking #184842595

## What does this PR do?

This PR is for sending notifications to a user and accommodation owner after booking a reservation.

## Description of Task to be completed?

1. Send emails to a user with details about the facility he/she has booked.
2. Sending in-app notifications to an accommodation manager after a user has booked with details of a user. 

## How should this be manually tested?

Book an accommodation with this route: POST http://localhost:<port>/api/v1/booking/:roomId

1. Check the email of the user after booking an accommodation
2. Use the route: GET http://localhost:<port>/API/v1/notification/, to check if your notification was saved

**## What are the relevant pivotal tracker/Trello stories?**
Notifications After Booking
**Screenshots**
On route: _GET http://localhost:<port>/API/v1/notification/_  
Success
![image](https://user-images.githubusercontent.com/52489534/230035490-87eb6556-4e38-4b01-84a8-1d6f00ba3a3f.png)
